internal/pkg/manifest/validate.go

```diff
@@ -843,6 +853,18 @@ func (ip IPNet) validate() error {
 
 // validate returns nil if NetworkLoadBalancerConfiguration is configured correctly.
 func (c NetworkLoadBalancerConfiguration) validate() error {
+	if err := c.MainListener.validate(); err != nil {
+		return fmt.Errorf("validate nlb.listener; %s", err)
```

Comment:
the `; %s` is here as well :) we need to fix them all to:
```suggestion
		return fmt.Errorf("validate nlb: %w", err)
```